modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -53,10 +53,11 @@ export default {
 <style lang="scss" scoped>
   .apos-input-wrapper--with-prefix {
     @include apos-input();
-
-    display: flex;
-    align-items: center;
-    color: var(--a-base-4);
+    & {
```

Comment:
the linter should complain about the missing empty line

Reply:
Fixed